### PR TITLE
[Outlook] (cleanup) Fix issues with 1.7 summary

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[ebd87d4](https://github.com/OfficeDev/office-js-docs-pr/pull/833/commits/ebd87d42adf51dc1c9a53e9ab9dd5e0beac7b658)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cleanup-1.7/docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201903201658129745-833%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=B0AmkjVojXyB964smsTmk%2FY1HtSQj%2BwyTR9zUAr0vzU%3D&st=2019-03-20T16%3A54%3A28Z&se=2019-04-20T16%3A59%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ Requirement set 1.7 includes all of the features of [Requirement set 1.6](../req
 - Added [RecurrenceTimeZone](/javascript/api/outlook_1_7/office.recurrencetimezone): Adds a new object that represents the time zone configuration of the recurrence pattern.
 - Added [SeriesTime](/javascript/api/outlook_1_7/office.seriestime): Adds a new object that provides methods to get and set the dates and times of appointments in a recurring series and to get the dates and times of meeting requests in a recurring series.
 - Added [Office.context.mailbox.item.addHandlerAsync](office.context.mailbox.item.md#addhandlerasynceventtype-handler-options-callback): Adds a new method that adds an event handler for a supported event.
-- Modified [Office.context.mailbox.item.from](office.context.mailbox.item.md#from-emailaddressdetailsfrom): Modifies to get the from value in Compose mode.
-- Modified [Office.context.mailbox.item.organizer](office.context.mailbox.item.md#organizer-emailaddressdetailsorganizer) - Modifies to get the organizer value in Compose mode.
+- Modified [Office.context.mailbox.item.from](office.context.mailbox.item.md#from-emailaddressdetailsfrom): Adds ability to get the from value in Compose mode.
```
Comment (AlexJerabek) [inline docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md]: Consider "Adds the ability"

File: docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ Requirement set 1.7 includes all of the features of [Requirement set 1.6](../req
...
+- Modified [Office.context.mailbox.item.organizer](office.context.mailbox.item.md#organizer-emailaddressdetailsorganizer): Adds ability to get the organizer value in Compose mode.
```
Comment (AlexJerabek) [inline docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md]: Consider "Adds the ability"

File: docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Requirement set 1.7 includes all of the features of [Requirement set 1.6](../req
 - Added [Office.MailboxEnums.RecurrenceTimeZone](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetimezone): Adds a new enum that specifies the time zone applied to the recurrence.
 - Added [Office.MailboxEnums.RecurrenceType](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetype): Adds a new enum that specifies the type of recurrence.
 - Added [Office.MailboxEnums.WeekNumber](/javascript/api/outlook_1_7/office.mailboxenums.weeknumber): Adds a new enum that specifies the week of the month.
-- Modified [Office.EventType](/javascript/api/office/office.eventtype): Modifies to support RecurrenceChanged, RecipientsChanged, and AppointmentTimeChanged events through addition of `RecurrenceChanged`, `RecipientsChanged`, and `AppointmentTimeChanged` entries respectively.
+- Modified [Office.EventType](/javascript/api/office/office.eventtype): Adds support for RecurrenceChanged, RecipientsChanged, and AppointmentTimeChanged events.
```
Comment (AlexJerabek) [inline docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md]: I think we should either keep the code ticks or, better yet, add links to the new reference content for those events.

Comment (ElizabethSamuel-MSFT) [reply docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md]: @AlexJerabek We don't have any docs for those events yet so there's nothing to link to beside the eventtype enum. I'll add the ``.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[97e4fd0](https://github.com/OfficeDev/office-js-docs-pr/pull/833/commits/97e4fd0dbb2c7958d95a69bb44c25e328d8d3468)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cleanup-1.7/docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201903201811466637-833%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kNIhCCzfqNjEfRIJJIZhA7dTq4vBsW%2FB5%2BBj5Notr0o%3D&st=2019-03-20T18%3A08%3A05Z&se=2019-04-20T18%3A13%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**